vllm/model_executor/layers/resampler.py

```diff
@@ -172,7 +172,10 @@ def __init__(
         self.query = nn.Parameter(torch.zeros(self.num_queries, embed_dim))
         trunc_normal_(self.query, std=0.02)
         if kv_dim is not None and kv_dim != embed_dim:
-            self.kv_proj = ReplicatedLinear(kv_dim, embed_dim, bias=False)
+            self.kv_proj = ReplicatedLinear(kv_dim,
+                                            embed_dim,
+                                            bias=False,
+                                            quant_config=quant_config)
```

Comment:
Could you pass in prefix here? Shouldn't this be covered by that case within the bitsandbytes linear method function?

Reply:
Done.  It looks bitsandbytes  doesn't handle `ReplicatedLinear` correctly